pieforproviders/pieforproviders
# [#1034] Adds hours attended field to the dashboard

# 💅🏼 What issue does this fix?
#1034 and #1033

## 🛷 Deployment


## 🧵 Steps to set up locally

`bundle exec rails db:migrate`

## 🧳 Steps to test
- try to process a spreadsheet in the old format (without Hours Attended) - it should work without throwing errors
- try to process a spreadsheet in the new format (with Hours Attended) - it should work without throwing errors
- the front end should load correctly without throwing errors
- when you hit the api for /case_list_for_dashboard hours_attended should be present in the response
